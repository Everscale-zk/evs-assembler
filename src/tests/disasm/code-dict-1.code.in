.fragment foo, {
    PUSHINT 0
    PUSHINT 1
    ADD
}
DICTPUSHCONST 32
DICTUGETJMPZ
.code-dict-cell 32, {
    x00000007 = foo,
}
THROW 78
