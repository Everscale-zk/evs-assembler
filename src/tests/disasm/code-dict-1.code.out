DICTPUSHCONST 32
.cell { ;; #6be5690a310c6cc1f557f54ef79dd0e3b0edee1b85515d992b4a887d8f10d62a
  .blob xa000000007
  ;; method 7
  PUSHINT 0
  PUSHINT 1
  ADD
}
DICTUGETJMPZ
THROW 78
